charts/pulsar/templates/function-worker-configmap.yaml

```diff
@@ -29,4 +29,6 @@ metadata:
     component: {{ .Values.functions.component }}
 data:
   pulsarDockerImageName: "{{ template "pulsar.imageFullName" (dict "image" .Values.images.functions "root" .) }}"
+{{ toYaml .Values.functions | indent 2 }}
```

Comment:
This doesn't look like the right solution. In this helm chart, the function worker runs together with the broker. The configuration in `.Values.broker.configData` with `PF_` prefix would get applied to the function_worker.yml file.

The keys in this file get applied directly in code with a timer, ever 5 minutes with this code https://github.com/apache/pulsar/blob/613a77100226628d8685d34260685d2df2b405ae/pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/kubernetes/KubernetesRuntimeFactory.java#L408-L422 to the KubernetesRuntimeFactory instance. 
The solution is buggy and will only work for String fields. Another problem is that initially it will start with the default values and override after running for 5 minutes. Therefore it seems like a useless solution.